tracee-rules/input.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"bufio"
+	"encoding/gob"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/aquasecurity/tracee/tracee-rules/types"
+)
+
+func setupStdinSource(inputSource string) (chan types.Event, error) {
+	res := make(chan types.Event)
+	scanner := bufio.NewScanner(os.Stdin)
+	go func() {
+		for scanner.Scan() {
+			event := scanner.Bytes()
+			switch inputSource {
+			case "tracee":
+				var e types.TraceeEvent
+				err := json.Unmarshal(event, &e)
+				if err != nil {
+					log.Printf("invalid json in %s: %v", string(event), err)
+				}
+				res <- types.Event(e)
+			}
+		}
+		close(res)
+	}()
+	return res, nil
+}
+
+func setupTraceeSource(traceeFilePath string) (chan types.Event, error) {
+	f, err := os.Open(traceeFilePath)
```

Comment:
Please add a test that the file is a "regular" file - I'm currently able to open a directory as a tracee file.
From current code:
```
	fi, err := os.Stat(inFilePath)
	if err != nil || !fi.Mode().IsRegular() {
		return nil, fmt.Errorf("Invalid Tracee input file path: %s", inFilePath)
	}
```